pandas/io/gbq.py

```diff
@@ -732,7 +742,7 @@ def generate_bq_schema(df, default_type='STRING'):
     return _generate_bq_schema(df, default_type=default_type)
 
 
-def _generate_bq_schema(df, default_type='STRING'):
+def _generate_bq_schema(df, default_type='STRING', default_mode='NULLABLE'):
```

Comment:
Can we remove `default_mode` as a parameter for now since we will only support the `'NULLABLE'` option?